src/app.ts

```diff
@@ -170,7 +171,7 @@ export function newApp(
   // Setup routes
   //
 
-  return tryCatch(async () => {
+  return new Task(async () => {
```

Comment:
we must ensure that this code never throws

Reply:
Are all static methods inside the Task that should never throw. I think that `tryCatch` before and `new Task` now, are only for chaining purposes with `withSpid`.